src/BlazorWasmSdk/Targets/Microsoft.NET.Sdk.BlazorWebAssembly.6_0.targets

```diff
@@ -169,7 +163,23 @@ Copyright (c) .NET Foundation. All rights reserved.
     <Message Importance="High" Text="$(MSBuildProjectName) (Blazor output) -&gt; $(TargetDir)wwwroot" Condition="'$(CopyBuildOutputToOutputDirectory)' == 'true' and '$(SkipCopyBuildProduct)'!='true'" />
   </Target>
 
-  <!-- Publish starts here -->
+  <Target
```

Comment:
Shoudn't this also be in the Wasm SDK?

Reply:
We should hopefully not need this target.

I am hoping we can express everything in terms of patterns, which makes things more declarative.